packages/react-components/react-swatch-picker-preview/stories/SwatchPicker/SwatchPickerLayout.stories.tsx

```diff
@@ -17,61 +19,83 @@ const useStyles = makeStyles({
   { color: '#FFC12E', value: 'FFC12E', 'aria-label': 'orange' },
-  { color: '#FEFF37', value: 'FEFF37', 'aria-label': 'yellow' },
   { color: '#90D057', value: '90D057', 'aria-label': 'light green' },
   { color: '#00B053', value: '00B053', 'aria-label': 'green' },
   { color: '#00AFED', value: '00AFED', 'aria-label': 'light blue' },
   { color: '#006EBD', value: '006EBD', 'aria-label': 'blue' },
-  { color: '#011F5E', value: '011F5E', 'aria-label': 'dark blue' },
-  { color: '#712F9E', value: '712F9E', 'aria-label': 'purple' },
 ];
 
+const images: ImageSwatchProps[] = [
+  {
+    src: 'https://fabricweb.azureedge.net/fabric-website/assets/images/swatch-picker/sea-swatch.jpg',
+    value: 'sea',
+    'aria-label': 'sea',
+  },
+  {
+    src: 'https://fabricweb.azureedge.net/fabric-website/assets/images/swatch-picker/bridge-swatch.jpg',
+    value: 'bridge',
+    'aria-label': 'bridge',
+  },
+  {
+    src: 'https://fabricweb.azureedge.net/fabric-website/assets/images/swatch-picker/park-swatch.jpg',
+    value: 'park',
+    'aria-label': 'park',
+  },
+];
+
+const items: SwatchProps[] = [...colors, ...images];
+
 export const SwatchPickerLayout = () => {
   const [selectedValue, setSelectedValue] = React.useState('00B053');
-  const [selectedColor, setSelectedColor] = React.useState('#00B053');
+  const [selectedSwatch, setSelectedSwatch] = React.useState('#00B053');
   const handleSelect: SwatchPickerOnSelectEventHandler = (_, data) => {
     setSelectedValue(data.selectedValue);
-    setSelectedColor(data.selectedColor);
+    const swatch = items.find(item => item.value === data.selectedValue) || items[0];
+    const src = (swatch as ImageSwatchProps).src;
+    if (swatch.color) {
+      setSelectedSwatch(swatch.color);
+    } else if (src) {
+      setSelectedSwatch(`url(${src}`);
+    }
   };
 
   const styles = useStyles();
-
   return (
     <>
       <h3>Row</h3>
-      <SwatchPicker aria-label="SwatchPicker default" selectedValue={selectedValue} onSelectionChange={handleSelect}>
+      <SwatchPicker aria-label="SwatchPicker row layout" selectedValue={selectedValue} onSelectionChange={handleSelect}>
         {colors.map(color => {
-          return <ColorSwatchWithTooltip key={color.value} {...color} />;
+          return <ColorSwatch key={color.value} {...color} />;
         })}
       </SwatchPicker>
       <h3>Grid</h3>
       <SwatchPicker
-        grid
-        aria-label="SwatchPicker default"
+        layout="grid"
+        aria-label="SwatchPicker grid layout"
         selectedValue={selectedValue}
         onSelectionChange={handleSelect}
       >
-        {renderSwatchPickerGrid(colors, 3, (row, index) => renderSwatchPickerRow(row, index, ColorSwatchWithTooltip))}
+        {renderSwatchPickerGrid({
+          items,
+          columnCount: 3,
+          renderRow: ({ children, rowId }) => <SwatchPickerRow key={rowId}>{children}</SwatchPickerRow>,
+          renderSwatch: (item: SwatchProps) =>
```

Comment:
isn't it the same as a default function?

Reply:
Yeah, I'll remove it